### ISSUE TITLE
Add Git filepath metadata into pmap event

### ISSUE THREAD

**Issue by capri-xiyue** [abcxyz/pmap#91]
### TL;DR

Add Git filepath metadata into pmap event(https://github.com/abcxyz/pmap/blob/41c46ce082d64d08396dd60bb3b5a48762f08bdd/apis/v1alpha1/pmap_event.pb.go#L123.)

Will need to change the logic in https://github.com/abcxyz/pmap/blob/main/.github/workflows/snapshot-file-copy.yml (currently it only records the relative path info instead of the whole file path when the file is uploaded to gcs), one idea is to copy the logic in https://github.com/abcxyz/pmap/blob/41c46ce082d64d08396dd60bb3b5a48762f08bdd/.github/workflows/snapshot-file-change.yml#L42 to copy all files under specific path to a specific folder with `--parents` option so that we can have the original git filepath info when we upload files to gcs bucket

**Comment by capri-xiyue** [comment]
Closed this one as it is a duplicate of https://github.com/abcxyz/pmap/issues/94
